addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -270,7 +268,7 @@ def _prepare_purchase_order_line(self, name, product_qty=0.0, price_unit=0.0, ta
         return {
             'name': name,
             'product_id': self.product_id.id,
-            'product_uom_id': self.product_id.uom_po_id.id,
+            'product_uom_id': self.product_id.uom_id.id,
```

Comment:
This would also have to change to `self.product_uom_id` if PO agrees with my previous comment about no need to convert blanket order UoMs to purchase uom anymore